i18n(de): Update `styling.mdx` translation (#4593)
withastro/docs: src/content/docs/de/guides/styling.mdx

Alle Astro-Stylingmethoden sind für eine [Markdown-Layoutkomponente](/de/core-concepts/layouts/#markdown-layouts) verfügbar.

Du kannst globale Styles auf deinen Markdown-Inhalt anwenden, indem du [`<style>`-Tags](#globale-styles) und [importierte Stylesheets](#externe-styles) in das Layout einfügst, das deinen Seiteninhalt umhüllt. Du kannst auch [CSS-Integrationen](#css-integrationen) einschließlich [Tailwind](/de/guides/integrations-guide/tailwind/) hinzufügen.

Wenn du Tailwind verwendest, kann das [Typografie-Plugin](https://tailwindcss.com/docs/typography-plugin) für das Styling von Markdown nützlich sein.

## Produktion

### Bündelungs-Kontrolle

Wenn Astro deine Seite für die Produktions-Veröffentlichung erstellt, minimiert es dein CSS und kombiniert es in Chunks. Jede Seite deiner Website erhält einen eigenen Chunk. Außerdem wird CSS, das von mehreren Seiten gemeinsam genutzt wird, zur Wiederverwendung in eigene Chunks aufgeteilt.

Wenn jedoch mehrere Seiten Styles teilen können einige gemeinsam genutzte Chunks sehr klein werden. Wenn alle separat gesendet würden, würde dies zu vielen Stylesheet-Anfragen führen und die Leistung der Website beeinträchtigen.

> Daher verlinkt Astro standardmäßig nur diejenigen in Ihrem HTML-Code mit einer Größe von mehr als 4 KB als `<link rel="stylesheet">`-Tags, während kleinere in `<style type="text/css">` eingefügt werden. Dieser Ansatz sorgt für ein Gleichgewicht zwischen der Anzahl zusätzlicher Anfragen und der Menge an CSS, die zwischen den Seiten zwischengespeichert werden kann.

Reviewer: ```suggestion
Daher verlinkt Astro standardmäßig nur Styles größer als 4 KB in deinem HTML-Code mit `<link rel="stylesheet">`-Tags, während kleinere in `<style type="text/css">` eingefügt werden. Dieser Ansatz sorgt für ein Gleichgewicht zwischen der Anzahl zusätzlicher Anfragen und der Menge an CSS, die zwischen den Seiten zwischengespeichert werden kann.
```

> Du kannst die Größe (in Bytes), bei der Stylesheets extern verlinkt werden, mithilfe der Vite-Build-Option `assetsInlineLimit` konfigurieren. Beachte, dass sich diese Option auch auf das Inlining von Skripten und Bildern auswirkt.

Reviewer: ```suggestion
Du kannst die Größe (in Bytes), ab der Stylesheets extern verlinkt werden, mithilfe der Vite-Build-Option `assetsInlineLimit` konfigurieren. Beachte, dass sich diese Option auch auf das Inlining von Skripten und Bildern auswirkt.
```

```js
import { defineConfig } from 'astro/config';

export default defineConfig({
  vite: {
    build: {
      assetsInlineLimit: 1024,
    }
  };
});
```

Wenn du möchtest, dass alle Projektstile extern bleiben, kannst du die Build-Option `inlineStylesheets` konfigurieren.

```js
import { defineConfig } from 'astro/config';

export default defineConfig({
  build: {
    inlineStylesheets: 'never'
  }
});
```

> Sie können diese Option auch auf `'always'` setzen, wodurch alle Stylesheets inline werden.

Reviewer: ```suggestion
Du kannst diese Option auch auf `'always'` setzen, wodurch alle Stylesheets direkt in den HTML-Code eingebettet werden.
```

> ## Fortgeschritten

Reviewer: ```suggestion
## Fortgeschrittene Anwendungsfälle
```

Ist zwar im englischen Original auch nicht vorhanden, aber einfach nur "Fortgeschritten" liest sich m.E. nicht gut.
